data/rubyday/rubyday-2016/videos.yml

```
@@ -140,8 +155,20 @@
     - Xavier Noria
   event_name: "rubyday 2016"
   date: "2016-11-25"
-  published_at: "TODO"
+  published_at: "2016-12-01"
   kind: "keynote"
   description: ""
   video_provider: "youtube"
   video_id: "U39Ou_eBkr4"
+
+- title: "Architecture Matters! A Trailblazer Workshop"
+  raw_title: "RubyDay2016 - Nick Sutterer - Architecture Matters! A Trailblazer Workshop"
+  speakers:
+    - Nick Sutterer
+  event_name: "rubyday 2016"
+  date: "2016-11-25"
+  published_at: "2016-12-01"
+  kind: "workshop"
+  description: ""
+  video_provider: "youtube"
+  video_id: "TODO"
```

If this wasn't recorded you can put it as:
```suggestion
  video_provider: "not_recorded"
  video_id: "nick-sutterer-workshop-rubyday-2016"
```